website/templates/base.html

```diff
@@ -122,6 +122,15 @@
                             <label id="description-error" class="error" for="description">{{form.description.errors}}</label>
                         {% endif %}
                     </div>
+                    <select name="label" class="form-control">
+                        <option value="0" selected="selected">Bug Type</option>
+                        <option value="1">Number error</option>
+                        <option value="2">Functional</option>
+                        <option value="3">Performance</option>
+                        <option value="4">Security</option>
+                        <option value="5">Typo</option>
+                        <option value="6">Design</option>
+                    </select>
```

Comment:
{{form.label}} does not render anything in base.html. I did it in a way similar to the one in issue_edit.html. It took me a lot of time trying to debug, but all in vain. Can we merge this and fix the base.html in a different commit. I want to work on issue_edit.html page but couldn't, as it would result in conflicts unless this gets merged.

Or else, please help me fix this.

Thanks :)